### PR TITLE
PhishMe integration enhancement and fix

### PR DESCRIPTION
**Pull request by Itay4**
Fixes https://github.com/demisto/etc/issues/10310

In addition, fixes extracted string command bug.

### CODE REVIEW

File: Integrations/integration-PhishMe.yml [hunk 1]
```diff
@@ -312,6 +311,18 @@ script:
         }
     }
 
+    function getReport(threatID) {
+        var endpoint = 't3/malware/' + threatID + '/pdf';
+        var report = sendRequest('GET', endpoint);
+        var fileName = 'Threat ID ' + threatID + ' report.pdf';
+        return {
+            Type: 3,
```
Comment (anara123) [inline Integrations/integration-PhishMe.yml]: use entryTypes instead

File: Integrations/integration-PhishMe.yml [hunk 2]
```diff
@@ -457,4 +471,11 @@ script:
...
     description: Check if sender address is malicious
-hidden: false
+  - name: phishme-get-report
+    arguments:
+    - name: threat-id
+      required: true
+      default: true
+      description: Thread ID to retrieve report of.
+    description: 'Retrieves individual PhishMe Intelligence reports.'
+releaseNotes: 'Added new command to retrieve reports from PhishMe'
```
Comment (anara123) [inline Integrations/integration-PhishMe.yml]: include the name of the command in the releaseNotes `Added new command "phishme-get-report" to ....`

File: Integrations/integration-PhishMe.yml [hunk 3]
```diff
@@ -138,42 +141,38 @@ script:
...
+            };
         } else {
-            md += "There are no results for this search\n";
-            ec = {"PhishMe" : {}};
-            ec.PhishMe = {"String": str, "NumOfThreats": 0};
+            md += 'No information found for this string';
         }
 
-        return ( {'ContentsFormat': formats.json, 'Type': entryTypes.note, 'Contents': res, "HumanReadable": md, "EntryContext": ec} );
+        return ( {'ContentsFormat': formats.json, 'Type': entryTypes.note, 'Contents': res, 'HumanReadable': md, 'EntryContext': ec} );
```
Comment (anara123) [inline Integrations/integration-PhishMe.yml]: write this in following way:
```
return {
    ContentsFormat: formats.json,
    Type: entryTypes.note,
    ....
}

```

**Comment by anara123** [comment]
@Itay4 also execute the command in the warroom and upload here the screenshot of the human readable.

**Comment by Itay4** [comment]
Screenshot of extracted string command:
screen shot 2018-01-21 at 9 55 28

**Comment by dorsha** [comment]
@itaimajlin what is the status of this PR?

**Comment by itaimajlin** [comment]
@dorsha wrong itai (: 
@Itay4

**Comment by dorsha** [comment]
Sorry :)

**Pull request closed without merging**